lib/features/manage_account/data/network/manage_account_api.dart

```diff
@@ -142,4 +146,40 @@ class ManageAccountAPI {
 
     return result?.list.first.rules ?? <TMailRule>[];
   }
+
+  Future<List<TMailRule>> createNewEmailRuleFilter(
```

Comment:
use the same API from PR: #824

Reply:
IMO, we should separate it so it's easy to change if `jmap` supports `create\delete` later. What do you mean?

Reply:
Ok, now we should sync all of them, if there are changes later, we can update the data layer again